com.cubrid.cubridmigration.core/src/com/cubrid/cubridmigration/core/engine/MigrationDirAndFilesManager.java

```diff
@@ -80,6 +81,51 @@ public String getFileName() {
         }
     }
 
+    /**
+     * Saves the parent, child depth, and number of files of the LOB directory
+     *
+     * @author Dongmin Kim
+     */
+    static class LobDirPath {
+        private static final int LOB_DIR_COUNT_MAX = 65000;
+        private static final int LOB_FILE_COUNT_MAX = 100000;
+
+        private long parentPath;
+        private int childPath;
+        private int lobFileCount;
+
+        private LobDirPath() {
+            this.parentPath = 1;
+            this.childPath = 1;
+            this.lobFileCount = 1;
+        }
+
+        /*
+         * Returns the directory depth by combining the parent and child.
+         * Example) 1/1, 34/1569, 129/64968
+         */
+        public String getLobDirDepth(String schemaName, String tableName) {
```

Comment:
It seems that the argument is not used.

Reply:
That was truly a meaningless task.
I will make the correction. Thank you for the review.